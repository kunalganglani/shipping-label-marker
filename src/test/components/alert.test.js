import React from 'react';
import renderer from 'react-test-renderer';
import Alert from '../../core/components/wizard/alert/alert';
test('Alert renders correctly', () => {
  const handleClose = jest.fn();
  const isError = true;
  const tree = renderer
    .create(<Alert onClose={handleClose} severity={isError ? 'error' : 'success'}></Alert>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
