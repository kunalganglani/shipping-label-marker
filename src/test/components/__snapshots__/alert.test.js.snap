// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
<div
  className="MuiPaper-root MuiAlert-root MuiAlert-filledError MuiPaper-elevation6"
  role="alert"
>
  <div
    className="MuiAlert-icon"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
      />
    </svg>
  </div>
  <div
    className="MuiAlert-message"
  />
  <div
    className="MuiAlert-action"
  >
    <button
      aria-label="Close"
      className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall"
      disabled={false}
      onBlur={[Function]}
      onClick={[MockFunction]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      title="Close"
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
          />
        </svg>
      </span>
    </button>
  </div>
</div>
`;
